src/vs/workbench/contrib/chat/browser/chatInputPart.ts

```diff
@@ -1515,7 +1530,9 @@ export class ChatInputPart extends Disposable implements IHistoryNavigationWidge
 	}
 
 	saveState(): void {
-		this.saveCurrentValue(this.getInputState());
+		const inputState = this.getInputState();
+		inputState.chatContextAttachments = inputState.chatContextAttachments?.filter(attachment => !attachment.isImage);
+		this.saveCurrentValue(inputState);
 		const inputHistory = [...this.history];
 		this.historyService.saveHistory(this.location, inputHistory);
```

Comment:
Do the attachments need to be filtered out of input history at this point, or are they already filtered here?